[charts] Unify legend customization hooks by exposing mark/gradient sub-classes on legend classes (#22533)
mui/mui-x: docs/data/charts/legend/legend.md

You can change the `label` style by targeting the root component's font properties.

> The mark is an SVG element, so you can use `legendClasses.markColored` (or `.MuiChartsLegend-markColored`) to change its color.

Reviewer: ```suggestion
The mark is an SVG element.
Use the `legendClasses.markColored` to target the element with `fill attribute.
```
